.github/CONTRIBUTING.md

```diff
@@ -129,26 +129,28 @@ See [Contributing to documentation related to PowerShell](#contributing-to-docum
   New changes always go into the **Unreleased** section.
   Keeping the changelog up-to-date simplifies the release process for Maintainers.
   An example:
-    ```
-    Unreleased
-    ----------
+  ```
```

Comment:
I'm fine going through the file completely and making it lint compliant, but I thought I was supposed to hold formatting changes to another PR so as to make the review easier😉  

I'll start another commit now. If you want me to drop it to another PR, I can, but it looks like everyone here is approving of the content change already.